src/utils/column-helper.ts

```diff
@@ -8,22 +9,23 @@ import { camelCase, deCamelCase, id } from '../utils';
  * @param {any[]} columns
  * @returns
  */
-export function setColumnDefaults(columns: any[]) {
+export function setColumnDefaults(columns: TableColumn[]) {
   if(!columns) return;
 
   for(let column of columns) {
     if(!column.$$id) {
       column.$$id = id();
     }
 
```

Comment:
Ok, I think it's better design to put the valueGetter on TableColumn instead since properties like `$$id` are already calculated in `setColumnDefaults()`